main.go

```diff
@@ -30,116 +32,116 @@ type FirebaseConfig struct {
 	Debug         bool
 }
 
-func NewFirebaseConfig() (*FirebaseConfig, error) {
-	empty := &FirebaseConfig{}
+func newFirebaseConfig() (*firebaseConfig, error) {
+	empty := &firebaseConfig{}
 
-	gcloud_user := GetOptionalEnv(GCLOUD_USER)
-	gcloud_project := GetOptionalEnv(GCLOUD_PROJECT)
+	gcloudUserValue := getOptionalEnv(gcloudUser)
+	gcloudProjectValue := getOptionalEnv(gcloudProject)
 
-	app_apk, err := GetRequiredEnv(APP_APK)
+	appApk, err := getRequiredEnv(appApk)
```

Comment:
not the best solution, as this will "shadow" the `appApk` const key in this function.

It works of course, but the const `appApk` would be better as ~ `envKeyAppAPK` or similar